packages/renderer/src/lib/dashboard/DashboardPage.svelte

```diff
@@ -1,18 +1,195 @@
 <script lang="ts">
-import { NavPage } from '@podman-desktop/ui-svelte';
+import { ListOrganizer, type ListOrganizerItem, NavPage, tablePersistence } from '@podman-desktop/ui-svelte';
+import { onMount } from 'svelte';
+import { SvelteMap } from 'svelte/reactivity';
 
-import { dashboardPageRegistry } from '/@/stores/dashboard/dashboard-page-registry.svelte';
+import {
+  convertFromListOrganizerItems,
+  dashboardPageRegistry,
+  type DashboardPageRegistryEntry,
+  defaultSectionNames,
+  setupDashboardPageRegistry,
+} from '/@/stores/dashboard/dashboard-page-registry.svelte';
 
 import NotificationsBox from './NotificationsBox.svelte';
+
+// Dashboard section configuration managed by dashboard page registry
+let dashboardSections = $state<ListOrganizerItem[]>([]);
+// eslint-disable-next-line svelte/no-unnecessary-state-wrap
+let dashboardOrdering = $state(new SvelteMap<string, number>());
```

Comment:
The SvelteMap does not provide reactivity on the assignment, since in other lines we re-assign the variable we lose the reactivity (that's how wrapping in a state is working) but this is not the right way

```suggestion
const dashboardOrdering = new SvelteMap<string, number>();
```

Reply:
Should be fixed